src/content/docs/it/tutorial/1-setup/3.mdx

```diff
@@ -0,0 +1,71 @@
+---
+type: tutorial
+title: Scrivi la tua prima riga di Astro
+description: |-
+  Tutorial: Costruisci il tuo primo blog Astro —
+  Fai le tue prime modifiche alla home page del tuo progetto tutorial
+---
+import Badge from '~/components/Badge.astro';
+import Checklist from '~/components/Checklist.astro';
+import Box from '~/components/tutorial/Box.astro';
+import InstallGuideTabGroup from '~/components/TabGroup/InstallGuideTabGroup.astro';
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro';
+import PreCheck from '~/components/tutorial/PreCheck.astro';
+
+
+
+<PreCheck>
+  - Fai le tue prime modifiche al tuo nuovo sito
+</PreCheck>
+
+## Modifica la tua home page
+
+1. Nel tuo editor di codice, naviga nella sezione Explorer a `src/pages/index.astro` e cliccaci sopra per aprire il contenuto del file in una scheda modificabile.
+
+    Il contenuto del tuo file `index.astro` dovrebbero apparire così:
+
+    ```astro title="src/pages/index.astro"
+    ---
+    ---
+
+    <html lang="en">
+      <head>
+        <meta charset="utf-8" />
+        <link rel="icon" type="image/svg+xml" href="/favicon.ico" />
+        <meta name="viewport" content="width=device-width" />
+        <meta name="generator" content={Astro.generator} >
+        <title>Astro</title>
+      </head>
+      <body>
+        <h1>Astro</h1>
+      </body>
+    </html>
+    ```
+
+2. Modifica il contenuto del `<body>` della tua pagina.
+
+    Inserisci nell'editor per modificare il testo del titolo sulla tua pagina e salva le modifiche.
+
+    ```astro title="src/pages/index.astro" del={2} ins={3}
+    <body>
+      <h1>Astro</h1>
+      <h1>My Astro Site</h1>
+    </body>
+    ```
+
+3. Controlla l'anteprima sul browser e dovresti vedere il contenuto della tua pagina aggiornato col nuovo testo.
+
+Congratulazioni! Ora sei uno sviluppatore Astro.
+
+Il resto di questa unità ti preparerà per il successo con il controllo di versione e un sito web pubblicato che puoi mostrare.
```

Comment:
This translation seems too "literal" to me. The original sentence is:
```
The rest of this unit will set you up for success with 
version control and a published website you can show off.
```
I would rephrase it to something more fluent, like:
```
Nel seguito di questa unità imparerai a utilizzare con successo 
i sistemi di controllo di versione e a pubblicare un sito web 
da mostrare al mondo.
```
Of course this is just a suggestion, feel free to adapt it as per your preferences.